slave.mk

```diff
@@ -278,6 +288,69 @@ endif
 export kernel_procure_method=$(KERNEL_PROCURE_METHOD)
 export vs_build_prepare_mem=$(VS_PREPARE_MEM)
 
+###############################################################################
+## Canned sequences
```

Comment:
maybe we should move this to Makefile.docker? all instructions related to docker should go there.

Reply:
I will make the change as you suggested and will get back to you.